infra/Pos/Reporting/Methods.hs

```diff
@@ -232,17 +238,25 @@ sendReport logFiles rawLogs reportType appName reportServerUri = do
     withSystemTempFile "main.log" $ \tempFp tempHandle -> liftIO $ do
         for_ rawLogs $ TIO.hPutStrLn tempHandle
         hClose tempHandle
+        rq0 <- liftIO (parseUrlThrow $ reportServerUri <//> "report")
```

Comment:
Hm, it seems all these `liftIO`s inside `do` are redundant because there is `liftIO` outside (3 lines above).